frontend/model/contracts/voting/proposals.js

```diff
@@ -59,16 +59,20 @@ function voteAgainst (state, { meta, data, contractID }) {
   sbp('okTurtles.events/emit', PROPOSAL_RESULT, state, VOTE_AGAINST, data)
 }
 
+// NOTE: The code is ready to receive different proposals settings,
+// However, we decided to make all settings the same, to simplify the UI/UX proptotype.
+const proposalDefaults = {
+  rule: RULE_PERCENTAGE,
+  expires_ms: 7 * DAYS_MILLIS,
```

Comment:
I noticed that the proposals have different expire days:
- Add/Remove Member proposals expire in 14 days. 
- Group Setting and Proposal Setting expire in 7 days.

Any reason for that? I suggest making them all the same, to simplify UI/UX as we did with the threshold.

cc @mmbotelho

Reply:
If we do, I would suggest keeping the upper limit of 14 days to give people more time!

Reply:
I'm okay with that.
Should we have a UI to change that setting too, or should we keep it "hidden" for now?

Reply:
I believe this is one of those things that we can set ourselves. We will find out if this timeframe works when the prototype is out and we can tweak it! I think adding a whole proposal to change this setting would be an overkill

Reply:
We probably need, however, to add a small text label indicating that a proposal is about to expire on the dashboard. Let me know what you think @sandrina-p @taoeffect and I can create an issue for it

Reply:
I like that idea, at least is informative. Later in the road, if can add create the UI/logic to update that info.